### PR TITLE
add multiple input elements

### PR DESCRIPTION
**Pull request by chris-kuehn**

### CODE REVIEW

File: components/Canvas.js [hunk 1]
```diff
@@ -3,67 +3,120 @@ import styled from "styled-components";
...
+    context.strokeStyle = `hsl(${color}, 100%, 50%)`;
+
+    for (let i = 0; i <= outline - 1; i++) {
+      let x = -60 - 20 * i;
+      let y = -60 - 20 * i;
+      let width = 120 + 40 * i;
+      let height = 120 + 40 * i;
+      context.lineWidth = variation[i];
+      context.strokeRect(x, y, width, height);
+    }
```
Comment (MerleMak) [inline components/Canvas.js:58]: this looks a bit too much like copied from the internet. Which is okay for complicated function but make sure to make it "yours" so you are able to understand it even in 2 years :)
I can't think of a good name for i :D
```suggestion
    for (let i = 0; i <= outline - 1; i++) {
      let coordinateOnXAxis = -60 - 20 * i;
      let coordinateOnYAxis = -60 - 20 * i;
      let widthOfRectangle = 120 + 40 * i;
      let heightOfRectangle = 120 + 40 * i;
      context.lineWidth = variation[i];
      context.strokeRect(x, y, width, height);
    }
```

File: components/Canvas.js [hunk 2]
```diff
@@ -3,67 +3,120 @@ import styled from "styled-components";
...
 
-    context.lineWidth = fourthLineWidth;
-    context.strokeRect(-100, -100, 200, 200);
+    // context.lineWidth = variation.thirdLineWidth;
+    // context.strokeRect(-80, -80, 160, 160);
+
+    // context.lineWidth = variation.fourthLineWidth;
+    // context.strokeRect(-100, -100, 200, 200);
 
     // Reset the transformation matrix to the identity matrix
```
Comment (MerleMak) [inline components/Canvas.js:69]: please delete the comments before committing your code:)

**Review by MerleMak — COMMENTED** [review]
Very nice job!
I left you two comments. I noticed in you deployment that your frame is too narrow. The labels and the inputs are not in the same line :)
Keep it going 🚀

File: components/Canvas.js (hunk 1 above)
Comment (chris-kuehn) [reply components/Canvas.js:58]: Hi Merle, 

I would just like to point out that I didn't copy the code or the artwork.

The artwork is very simple and not very sophisticated, I can do it without copying anything. ;)

The technique to draw several elements in canvas with one loop is a technique I learned in canvas tutorials. Also using the variable i to change the coordinates and the size of the rectangles on each loop is a technique I already knew before our bootcamp.

If I should copy things, which is not planned, then I will discuss this with you couches beforehand and mark it accordingly in my code as agreed.

But you are of course right, that "coordinateOnXAxis" is more self-explanatory than just "x".  Will be changed!!! ;)

Best regards and thanks for your review :)

Christian :)

File: components/Canvas.js (hunk 2 above)
Comment (chris-kuehn) [reply components/Canvas.js:69]: I have forgotten to delete it ... sorry :)

**Pull request closed without merging**